src/components/CrosshairToggle.jsx

```diff
@@ -9,7 +9,7 @@ const CrosshairToggle = ({
 }) => {
     const CrosshairIcon = [CrosshairOffIcon, CrosshairOnIcon, CrosshairTooltipIcon][state];
     return (
-        <div className="ciq-menu">
+        <div className={`ciq-menu ${(state !== 0) ? 'stxMenuActive' : ''}`}>
```

Comment:
`stxMenuActive` for toggle? That's very confusing. We styled this without needing `stxMenuActive`. Why is it needed here now?

Reply:
@brucebinary  as you see in [the design](https://app.zeplin.io/project/5b3d8a662f4775474e13e26f/screen/5b4f012de88a0ad55ec70460) when crosshair gets active, like other menus it should have the white top border. so why not using the existing classes besides defining new.

Reply:
@shayanbinary  better that you do this styling in CSS than in jsx

Reply:
@shayanbinary actually nvm that.

Reply:
@brucebinary  how about to handle it by another class name like `crosshair-active`? because its menu should have the top border while crosshair is active.

Reply:
@brucebinary As I discuss with Ashkan, crosshair has the wrong design, so I remove what I added here.